tox.ini

```diff
@@ -178,6 +187,7 @@ commands =
     {[testenv:docs]commands}
     {[testenv:etl]commands}
     {[testenv:scripts]commands}
+    {[testenv:unit]commands}
```

Comment:
I believe these are run in order, and since the unit tests run quickly, maybe it makes sense to put them before the ETL, so that if they pick up a problem, the tests fail quickly, rather than taking half an hour.